### PR TITLE
OperatorSource and supporting files

### PR DESCRIPTION
**Pull request by bmozaffa**
Providing the required files and symbolic links to build and upload an operator bundle to OpenShift, and install the operator by creating an OperatorSource
    
Signed-off-by: Babak Mozaffari <bmozaffa@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,28 @@ docker push quay.io/kiegroup/kie-cloud-operator:<version>
...
+Push the operator bundle to your quay application repository as follows:
+
+```bash
+operator-courier push deploy/catalog_resources/courier/bundle_dir kiegroup kiecloud-operator 1.0.0 "basic XXXXXXXXX"
+```
+
+If pushing to another quay repository, replace _kiegroup_ with your username or other namespace. Also note that the push command does not overwrite an existing repository, and it needs to be deleted before a new version can be built and uploaded. Once the bundle has been uploaded, create an [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster) to load your operator bundle in OpenShift.
+
+```bash
+oc create -f deploy/catalog_resources/courier/businessautomation-operatorsource.yaml
```
Comment (tchughesiv) [inline README.md]: this file has changed

Comment (bmozaffa) [reply README.md]: Updated

**Pull request merged**